script/eslint-plugin-va/rules/prefer-web-component-library.js

```diff
@@ -210,6 +210,93 @@ const paginationTransformer = (context, node) => {
     ],
   });
 };
+const tableTransformer = (context, node) => {
+  const componentName = node.openingElement.name;
+  const currentSortNode = getPropNode(node, 'currentSort');
+  const dataNode = getPropNode(node, 'data');
+  const fieldsNode = getPropNode(node, 'fields');
+  const dataValue = dataNode?.value.expression || dataNode?.value;
+  const fieldsValue = fieldsNode?.value.expression || fieldsNode?.value;
+  const sourceCode = context.getSourceCode();
+  const importNode = context
+    .getAncestors()[0]
+    .body.find(
+      node =>
+        node.type === 'ImportDeclaration' &&
+        node.source.value.includes(
+          `@department-of-veterans-affairs/component-library/${
+            componentName.name
+          }`,
+        ),
+    );
+
+  context.report({
+    node,
+    message: MESSAGE,
+    data: {
+      reactComponent: componentName.name,
+      webComponent: 'va-table',
+    },
+    suggest: [
+      {
+        desc: 'Migrate component',
+        fix: fixer => {
+          return [
+            // Import utility function that can easily handle the existing props
+            // for the React version of `<Table>`
+            fixer.replaceText(
+              importNode.specifiers[0].local,
+              `{ generateTableChildren }`,
+            ),
+
+            // Remove Table name from import path
+            fixer.replaceText(
+              importNode.source,
+              `'@department-of-veterans-affairs/component-library'`,
+            ),
+
+            // Rename component tag
+            fixer.replaceText(componentName, 'va-table'),
+
+            // Remove the data prop
+            fixer.remove(dataNode),
+
+            // Remove the fields prop
+            fixer.remove(fieldsNode),
+
+            // Remove the currentSort prop
+            currentSortNode && fixer.remove(currentSortNode),
+
+            // Add `sort-column` prop
+            currentSortNode &&
+              fixer.insertTextBefore(currentSortNode, 'sort-column="0"'),
+
+            // Add `descending` prop
+            currentSortNode &&
+              fixer.insertTextBefore(currentSortNode, ' descending'),
```

Comment:
This is very simple, and the `sort-column` prop might be for the wrong index and `descending` may not be necessary. I don't think there's a good way for me to interpret the code that's being analyzed, so I'm just adding these as a "kitchen sink" example so the engineer doing the migration knows how to work with sorting.